### ISSUE TITLE
Method to export a drifter dataset to formats that can be viewed in GIS tools

### ISSUE THREAD

**Issue by knutfrode** [OpenDrift/trajan#90]
Would be nice to have

**Comment by gauteh** [comment]
Shapefiles? Geojson? I think geopandas can help us.

**Comment by knutfrode** [comment]
Yes, I also think GeoPandas is great, and we should not reinvent any wheels.
But it is not obvious what is best format to use for trajectories / particles.

**Comment by gauteh** [comment]
:+1: I used geopandas before to create shapefiles for arcgis with aggregated data in areas.
